src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "countrys")
```

Comment:
The table name is annotated as `countrys`, which is likely a typo. It should probably be `countries` to follow standard naming conventions for pluralization.